django/src/rdwatch_scoring/views/vector_tile.py

```diff
@@ -135,17 +135,18 @@ def vector_tile(
             .values()
             .annotate(
                 id=F('uuid'),
+                timestamp=F('date'),
                 mvtgeom=mvtgeom,
                 configuration_id=F('site_uuid__evaluation_run_uuid'),
-                site_number=F('site_uuid__site_id'),
+                site_number=Substr(F('site_uuid__site_id'), 9),  # pos is 1 indexed
                 label=Case(
                     When(
                         Q(phase__isnull=False),
                         Lower(Replace('phase', Value(' '), Value('_'))),
                     ),
                     default=Value('unknown'),
                 ),  # This should be an ID, on client side can make it understand this
-                area=Area(Transform('transformedgeom', srid=6933)),
```

Comment:
This was my bad GitHub suggestion.  Didn't mean to overwrite this, this should be left in.